libcloud/test/compute/fixtures/dimensiondata/network_firewallRule.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<firewallRules xmlns="urn:didata.com:api:cloud:types" pageNumber="1" pageCount="13" totalCount="13" pageSize="50">     
+<firewallRules xmlns="urn:didata.com:api:cloud:types" pageNumber="1" pageCount="14" totalCount="14" pageSize="50">
```

Comment:
For the file I think you'll lose some coverage by removing those rules.  Rules are all over the place, I just added some more to make sure it at least was able to parse all of the different types of rules we have.

Also page count and total count are way off now with those deleted rules (probably were before)